app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::ReviewsController < ApplicationController
+
+  def index
+    render json: Review.all, adapter: :json
+  end
```

Comment:
This needs to be specified in a config file. See the serializer reading from Week 6